apps/full-site-editing/full-site-editing-plugin/site-editor/index.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Site Editor experiment file.
+ *
```

Comment:
Could we add some more comments explaining what this does vs what dotcom-fse does? possibly link back to Gutenberg too